README.md

```diff
@@ -44,14 +44,18 @@ xk6 build --with github.com/nspcc-dev/xk6-neofs=.
 
 Create native client with `connect` method. Arguments:
 - neofs storage node endpoint
-- WIF (empty value produces random key)
+- hex encoded private key (empty value produces random key)
```

Comment:
What's wrong with WIF?

Reply:
It seems to me the key is more convenient and clear for the external user

Reply:
Arguably, but ok.